module-system/sov-state/src/prover_storage.rs

```diff
@@ -79,11 +79,11 @@ impl<S: MerkleProofSpec> Storage for ProverStorage<S> {
             .map(|root| root.0)
     }
 
-    fn validate_and_commit(
+    fn calculate_state_root_and_node_batch(
```

Comment:
nit: This is name is a tiny bit verbose. Maybe something like "compute_state_update" would be a little easier to read?

Reply:
fixed